### ISSUE TITLE
scheduler car for automations only? [Feauture Request]

### ISSUE THREAD

**Issue by NiMa08** [nielsfaber/scheduler-card#383]
Hi Niels, love the scheduler card. Your gui is great and way better then HA's way of working with automations. it's a lot more user friendly. It would be great if it could be implemented to create automations without schedules. Is this in any way possible? And do you have a contribution page somewhere where i can contribute some beers. @nielsfaber

**Comment by nielsfaber** [comment]
Thanks for liking scheduler!
I agree 100% with you that HA isn't that intuitive when it comes to setting up an automation on the fly (e.g. from your phone).
Which I why I started this project 😄 

What would be the advantage of having scheduler generate automations?
Currently the scheduler-component integration is written completely independent from automations, they have little in common (besides the fact that both have a triggers/condition/actions structure.
So making this request happen is not an easy job.

On the bottom of the readme page there is a link to buy-me-a-beer. It is much appreciated, really special to see that people are willing to donate to a complete stranger (though a real beer would be even better!). Note that it is not a way to get your Feature Request done though..

And sorry for breaking scheduler with the last release (in case you noticed). Work in progress.

**Comment by NiMa08** [comment]
Alright. I thought it was written like automations. The advantage would be that you can set up an automation without a schedule, like turning on a tv lightstrip when the tv turns on. But then with your gui everyone could set up an automation. My gf for example 😅

**Comment by NiMa08** [comment]
What is the best way to ask a feature request

**Comment by NiMa08** [comment]
First timer ha

**Comment by nielsfaber** [comment]
I’m doing my best to keep scheduler simple as it is now actually.
My feeling is that it will quickly become difficult to use (and to maintain for me) if we would expand the capabilities.

The golden rule is that scheduler only can trigger an action on a certain time of the day.
It cannot wait for a device to change and act on that.

I understand that you would like to create simple automations with it, but I think this is a big task and something that doesn’t fit with my original idea behind scheduler.
Hope you can understand this.

**Comment by NiMa08** [comment]
Of course. No problem. One can hope. And i din’t know how you wrote the code. So thought it was easier then your saying. So i understand. Then hopefully HA will eventually get an update that makes it easoer
